api/install.py

```diff
@@ -186,7 +190,6 @@ def get_cls_name_helper(ss):
 - name: prob
   reference_name: label
   type: prob
-perf_metric:
```

Comment:
Are we willing to keep using this script at all? If we are, perf_metric should remain here, right?